db/migrate/20150805212906_add_fields_to_concepts.rb

```diff
@@ -1,6 +1,6 @@
 class AddFieldsToConcepts < ActiveRecord::Migration
   def change
     add_column :concepts, :parent_id, :integer
-    add_column :concepts, :uid, :string, null: false
+    add_column :concepts, :uid, :string
   end
```

Comment:
@marcellosachs This migration already ran on production, hope that's not a problem.